src/components/ExpandableInfo.tsx

```diff
@@ -1,216 +1,162 @@
   children?: React.ReactNode
   image?: IGatsbyImageData
   title: ReactNode
   contentPreview: ReactNode
-  background: string
-  forceOpen: boolean
+  background: BackgroundProps["background"]
+  forceOpen?: boolean
   className?: string
 }
 
-const ExpandableInfo: React.FC<IProps> = ({
-  image,
-  title,
-  contentPreview,
-  children,
-  background,
-  forceOpen,
-  className,
-}) => {
-  const [isVisible, setIsVisible] = useState(forceOpen)
-  const expandCollapse = {
-    collapsed: {
-      height: 0,
-      transition: {
-        when: "afterChildren",
-      },
-    },
-    expanded: {
-      height: "100%",
-      transition: {
-        when: "beforeChildren",
-      },
+const ExpandableInfo = forwardRef<IProps, "div">(
+  (
+    {
+      image,
+      title,
+      contentPreview,
+      children,
+      background,
+      forceOpen,
+      className,
+      ...rest
     },
-  }
-  const chevronFlip = {
-    collapsed: {
-      rotate: 0,
-      transition: {
-        duration: 0.1,
+    ref
+  ) => {
+    const { isOpen, getButtonProps, getDisclosureProps } = useDisclosure({
+      defaultIsOpen: forceOpen,
+    })
+
+    const chevronFlip = {
+      collapsed: {
+        rotate: 0,
+        transition: {
+          duration: 0.1,
+        },
       },
-    },
-    expanded: {
-      rotate: 180,
-      transition: {
-        duration: 0.4,
+      expanded: {
+        rotate: 180,
+        transition: {
+          duration: 0.4,
+        },
       },
-    },
-  }
-  const showHide = {
-    collapsed: {
-      display: "none",
-    },
-    expanded: {
-      display: "inline-block",
-    },
-  }
-  const fadeInOut = {
-    collapsed: {
-      opacity: 0,
-      transition: {
-        duration: 0.1,
-      },
-    },
-    expanded: {
-      opacity: 1,
-      transition: {
-        duration: 0.4,
-      },
-    },
-  }
-  return (
-    <Card background={background} className={className}>
-      <Content>
-        {image && <GatsbyImage image={image} alt="" />}
-        <TitleContent>
-          <Question>
-            <Header>
-              <Title>{title}</Title>
-            </Header>
-            <TextPreview>{contentPreview}</TextPreview>
-          </Question>
-        </TitleContent>
-        {!forceOpen && (
-          <ButtonContainer
-            variants={chevronFlip}
-            animate={isVisible ? "expanded" : "collapsed"}
-            initial={false}
-            onClick={() => setIsVisible(!isVisible)}
-          >
-            <Icon name="chevronDown" size="36" />
-          </ButtonContainer>
-        )}
-      </Content>
-      <motion.div
-        variants={expandCollapse}
-        animate={isVisible ? "expanded" : "collapsed"}
-        initial={false}
+    }
+
+    const animateToggle = isOpen ? "expanded" : "collapsed"
+
+    return (
+      <VStack
+        border="1px solid"
+        borderColor="border"
+        borderRadius="2px"
+        p="1.5rem"
+        mb="1rem"
```

Comment:
In general, lets try to use as much as we can the Chakra scale system (for font sizes & spacing). To keep it consistent with the rest of the codebase and future design system.

```suggestion
        p={6}
        mb={4}
```

Reply:
A bad habit of mine. I'll be sure to keep this in mind for future changes and any other existing PRs! 😄